java/KafkaConfigProviders/Kafka-mTLS-Keystore-Sql-ConfigProviders/.gitignore

```diff
@@ -0,0 +1 @@
+!*.jar
```

Comment:
I just noticed we do this in the other two examples with local jar, but this is too "aggressive" and remove the ignore also for the jar that will be generated by building this project.
In all 3 configprovider examples this gitignore should be `!local-repo/**/*.jar`  rather than just `!*.jar`
(please double check that the statement actually remove the ignore of the jars in the local-repo only; I haven't checked myself)

Reply:
Updated